test/extended/setup.sh

```diff
@@ -34,10 +34,7 @@ function os::test::extended::setup () {
 	export KUBE_REPO_ROOT="${OS_ROOT}/vendor/k8s.io/kubernetes"
 
 	os::util::environment::setup_time_vars
-	os::util::environment::use_sudo
-	os::util::environment::setup_all_server_vars "test-extended/core"
-
-	os::util::ensure::iptables_privileges_exist
+	os::util::environment::setup_tmpdir_vars "test-extended/core"
```

Comment:
Every time we `setup_all_server_vars` we also `setup_tmpdir_vars` ... and in this case, we would also be OK if we did the `setup_tmpdir_vars` twice (once here, once below) so we _should_ have `setup_tmpdir_vars` as a sub-task of `setup_all_server_vars`.